In [1]:
# Import Libraries
import numpy as np
from scipy import integrate

#### Cari Integral dari:

$f(3x^2 - 6x + 3 )dx$

In [2]:
# menetapkan range batas
x=np.linspace(3,6)

In [3]:
# mendeklasrasikan fungsi
y=3*x**2-6*x+3

In [4]:
# Integral dari fungsi
integrate.trapz(x,y)

296.9943773427739

integral $f(3x^2 - 6x + 3 )dx$ dengan range titik x 3 sampai dengan 6 adalah 296.99

#### Cari Integral dari:

$f(8x^3 - x^2 + 5x -1 )dx$

In [5]:
# menetapkan range batas
x=np.linspace(2,4)

In [6]:
# mendeklarasikan fungsi
y=8*x**3-1*x**2+5*x-1

In [7]:
# Integral dari fungsi
integrate.trapz(x,y)

1432.6272386505623

Integral $f(8x^3 - 1x^2 + 5x -1 )dx$ dengan range titik x 2 sampai dengan 4 adalah 1431.63